Certifications Consul Associate 003 Coming Soon (#2188)
hashicorp/dev-portal: src/content/certifications/exam-faqs/consul-associate-003.mdx

<!-- Each "## Heading Two" section in this document will be used to populate an FAQ item -->

## Exam Availability

Coming soon!

## Which Exam to Take

> The [Consul Associate 002 certification](https://developer.hashicorp.com/certifications/networking-automation#consul-associate-002-details) is still relevant and will be accepted as validation of Consul knowledge until each individual badge’s expiration date. Receiving a Consul Associate 003 certification will not impact any existing Consul Associate 002 credentials, and a single candidate can hold both at the same time. However, either certification alone can be used to validate Consul knowledge at the associate level.

Reviewer: since this is on devdot we don't need to include the full path

```suggestion
You can use either exams to validate Consul knowledge at the associate level. The [Consul Associate 002 certification](/certifications/networking-automation#consul-associate-002-details) is still relevant and will be accepted as validation of Consul knowledge until the badge’s expiration date. Receiving a Consul Associate 003 certification will not impact any existing Consul Associate 002 credentials. You can hold both at the same time.
```

**If you have studied for the Consul Associate 002 and are prepared to take it now,** schedule and take your 002 exam at your earliest convenience.

> **If you are not ready to schedule and take your exam until early 2024,** we advise studying for the 003 exam and taking that version of the exam after it launches in early 2024.

Reviewer: ```suggestion
**If you are not ready to schedule and take your exam until early 2024,** we advise studying for the 003 exam and taking that version of the exam when it launches in early 2024.
```
